enhancements/storage/csi-driver-install.md

```diff
@@ -108,6 +108,13 @@ Cons:
 2. In general there are some concerns about OLM's lack of documentation and everyone's understanding of it.
 3. Not sure about disconected cluster installation / update.
 
+Open questions for OLM team:
+1. How will disconnected installs work?
+2. We need a way for a CSI driver operator to say version range of Openshift against which it is supported.
+3. Are channel to which user is subscribed to automatically upgraded when Openshift version is bumped? For example: If we install an operator from 4.2 channel on OCP-4.2 and then upgrade to OCP-4.3, is subscription updated to use channel 4.3? Or this should be handled via `skipRange`?
+4. Currently CVO operators can directly access cloudprovider configuration via configmap placed in `openshift-config` namespace, are we going to allow OLM operators to do the same? Do we need to do something to support CSI driver configuration?
```

Comment:
I think it's out of scope of OLM and that that CSI operator(s) must handle by themselves (e.g. by having appropriate RBAC rules).